packages/rest/src/rest.server.ts

```diff
@@ -666,6 +667,24 @@ export class RestServer extends Context implements Server, HttpServerLike {
     );
   }
 
+  /**
+   * Register a route redirecting callers to a different URL.
+   *
+   * ```ts
+   * server.redirect('/explorer', '/explorer/');
+   * ```
+   *
+   * @param source URL path of the redirect endpoint
+   * @param target Location (URL path) where to redirect to
+   * @param statusCode HTTP status code to respond with,
+   *   defaults to 303 (See Other).
+   */
+  redirect(source: string, target: string, statusCode?: number): Binding {
```

Comment:
The names `source` and `target` sound a little weird. Maybe `fromUrl` and `toUrl` is more descriptive?

Reply:
Good point. Will use the following names:

```ts
class RestServer {
  //...

  redirect(
    fromPath: string,
    toPathOrUrl: string,
    statusCode?: number,
  ): Binding;
}
```